src/include/buffer.h

```diff
@@ -881,6 +881,13 @@ namespace buffer CEPH_BUFFER_API {
     }
     uint32_t crc32c(uint32_t crc) const;
     void invalidate_crc();
+
+    // These functions return a bufferlist with a pointer to a single
+    // static buffer. They /must/ not outlive the memory they
+    // reference.
+    static list static_from_chararray(char* c, size_t l);
```

Comment:
It's not intended for a static array necessarily. I just wanted a shortcut for if I happen to have a pointer and length.